owncloudApp/src/main/java/com/owncloud/android/shares/presentation/ShareActivity.kt

```diff
@@ -394,6 +403,14 @@ class ShareActivity : FileActivity(), ShareFragmentListener {
         fileOperationsHelper.copyOrSendPrivateLink(file)
     }
 
+    private fun updateSharedWithSharee(isSharedWithSharee: Boolean) {
+        val file = storageManager.getFileByPath(file.remotePath)
```

Comment:
why don't you do it in a better kotlin?? :)

```
storageManager.getFileByPath(file.remotePath)?.let {
    it.isSharedWithSharee = isSharedWithSharee
    storageManager.saveFile(it)
}
```

Reply:
Done